tests/test_specific_rules.py

```diff
@@ -152,6 +152,66 @@ def test_new_terms_fields_unique(self):
                     f"new terms fields values are not unique - {rule.contents.data.new_terms.value}"
 
 
+class TestEQLInSet(BaseRuleTest):
```

Comment:
nit: Can this test be added to any other existing test class?

Reply:
I do not think we have any existing test classes in Detection Rules that are directly running tests against EQL Queries? I am not sure where else would be applicable this test.

Reply:
Going to make a new test file to test specifically for python library test cases.

Reply:
Done